### ISSUE TITLE
Width: this method is only available for raster images

### ISSUE THREAD

**Issue by kmsydney** [hugomods/images#58]
When trying to use an SVG image, for example:

```
![Hugo](hugo-logo-wide.svg)
```

`hugomods/images` fails with the following error:

```
error calling partial: "/Users/kmsydney/Library/Caches/hugo_cache/modules/filecache/modules/pkg/mod/github.com/hugomods/images@v0.11.0/layouts/partials/images/image.html:73:29": execute of template failed: template: partials/images/image.html:73:29: 
executing "partials/images/image.html" at <$res.Width>: error calling Width: this method is only available for raster images. To determine if an image is SVG, you can do {{ if eq .MediaType.SubType "svg" }}{{ end }}
```

I expected that images mod would skip the Width check for svg images and only apply that logic to raster images.

**Comment by razonyang** [comment]
Should be fixed since [v0.11.1](https://github.com/hugomods/images/releases/tag/v0.11.1). Thanks for reporting this issue.

**Comment by razonyang** [comment]
Please upgrade to v0.11.2 instead, there is minor change about the warning messages.
